pkg/resources/dynamic_table_acceptance_test.go

```diff
@@ -25,13 +25,15 @@ func TestAcc_DynamicTable_basic(t *testing.T) {
 	tableName := name + "_table"
 	m := func() map[string]config.Variable {
 		return map[string]config.Variable{
-			"name":       config.StringVariable(name),
-			"database":   config.StringVariable(acc.TestDatabaseName),
-			"schema":     config.StringVariable(acc.TestSchemaName),
-			"warehouse":  config.StringVariable(acc.TestWarehouseName),
-			"query":      config.StringVariable(fmt.Sprintf(`select "id" from "%v"."%v"."%v"`, acc.TestDatabaseName, acc.TestSchemaName, tableName)),
-			"comment":    config.StringVariable("Terraform acceptance test"),
-			"table_name": config.StringVariable(tableName),
+			"name":         config.StringVariable(name),
+			"database":     config.StringVariable(acc.TestDatabaseName),
+			"schema":       config.StringVariable(acc.TestSchemaName),
+			"warehouse":    config.StringVariable(acc.TestWarehouseName),
+			"initialize":   config.StringVariable("ON_SCHEDULE"),
```

Comment:
These are both optional parameters. Please add the steps to show how they behave with changes (e.g. first create without them set, then set one of them, change it, set the other one).